packages/form-builder/src/blocks/fields/amount/inspector/index.js

```diff
@@ -7,11 +7,50 @@ import {CurrencyControl} from "../../../../common/currency";
 
 const Inspector = ({attributes, setAttributes}) => {
 
-    const {levels} = attributes;
+    const {
+        levels,
+        priceOption,
+        setPrice,
+        customAmount,
+        customAmountMin,
+        customAmountMax,
+    } = attributes;
 
     return (
         <InspectorControls>
-            <PanelBody title={__('Donation Levels', 'give')} initialOpen={true}>
+            <PanelBody title={__('Donation Options', 'give')} initialOpen={true}>
+                <SelectControl
+                    label={__('Donation Option', 'give')}
+                    onChange={(priceOption) => setAttributes({priceOption})}
+                    value={priceOption}
+                    options={[
+                        {label: __('Multi-level Donation', 'give'), value: 'multi'},
+                        {label: __('Fixed Donation', 'give'), value: 'set'},
+                    ]}
+                />
+                {priceOption === 'set' && (
+                    <CurrencyControl
+                        label={__('Set Donation', 'give')}
+                        value={setPrice}
+                        onValueChange={(setPrice) => setAttributes({setPrice})}
+                    />
+                )}
+            </PanelBody>
+            <PanelBody title={__('Custom Amount', 'give')} initialOpen={false}>
+                <ToggleControl
+                    label={__('Custom Amount', 'give')}
+                    checked={customAmount}
+                    onChange={() => setAttributes({customAmount: !customAmount})}
+                />
+                { !!customAmount && (
+                    <>
+                        <CurrencyControl label={__('Minimum', 'give')} value={customAmountMin} onValueChange={(value) => setAttributes({customAmountMin: value})} />
+                        <CurrencyControl label={__('Maximum', 'give')} value={customAmountMax} onValueChange={(value) => setAttributes({customAmountMax: value})} />
+                    </>
+                )}
+            </PanelBody>
+            {priceOption === 'multi' && (
+                <PanelBody title={__('Donation Levels', 'give')} initialOpen={false}>
```

Comment:
Something's off with the level styles:

<img width="285" alt="Screenshot 2023-01-31 at 10 01 54 AM" src="https://user-images.githubusercontent.com/10138447/215796658-9bf2b198-5cdb-4d97-83b3-b63ff2a350e7.png">

Reply:
Whoops. That seems to be a break when altering the underlying component. Resolved, see https://github.com/impress-org/givewp-next-gen/pull/119/commits/880e7393072d21c62d88dd17d263736e1d0a974f.

Reply:
Still a tiny bit of margin bottom left.  I've seen since we updated gutenberg theres this "StyledField-deprecatedMarginField" class that keeps messing with us.  Just targeting the first div child works here:

<img width="269" alt="Screenshot 2023-02-01 at 10 42 00 AM" src="https://user-images.githubusercontent.com/10138447/216090872-137cc981-2ba7-4760-9ea3-dfdb9ac76c00.png">

<img width="321" alt="Screenshot 2023-02-01 at 10 43 04 AM" src="https://user-images.githubusercontent.com/10138447/216090916-ff577266-b932-4233-8643-d98c5237b24e.png">

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/119/commits/345a72a480133effccf18d3902e355e3539e35b4.